packages/nx/src/tasks-runner/invoke-runner.ts

```diff
@@ -0,0 +1,54 @@
+import { workspaceConfigurationCheck } from '../utils/workspace-configuration-check';
+import { readNxJson } from '../config/configuration';
+import { NxArgs } from '../utils/command-line-utils';
+import { createProjectGraphAsync } from '../project-graph/project-graph';
+import { Task, TaskGraph } from '../config/task-graph';
+import { invokeTasksRunner } from './run-command';
+import { InvokeRunnerTerminalOutputLifeCycle } from './life-cycles/invoke-runner-terminal-output-life-cycle';
+import { performance } from 'perf_hooks';
+
+export async function invokeRunner(nxArgs: NxArgs) {
```

Comment:
Could we maybe call this "getTaskRunner" or similar? Invoke runner makes it sound like you are calling into it immediately,  whereas you need to call invokeRunner().invoke() to actually do it.

getTaskRunner().invoke() reads a lot nicer.